### ISSUE TITLE
Bug(prefer-namespace-import): fix should keep type imports

### ISSUE THREAD

**Issue by marcalexiei** [marcalexiei/eslint-plugin-zod-x#39]
### 🧩 Description

Keep import type when applying the fix

### 🪜 How to reproduce

import type { z } from 'zod';

### ✅ Expected behavior

import type * as  z from 'zod';

### 💻 System info

```shell
N/A
```

### ✒️ Additional context

- followup of #38
